### PR TITLE
feat: Pokedex with platformatic

### PR DESCRIPTION
**Pull request by Eomm**

### CODE REVIEW

File: posts/bonus-pokedex.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
+# How to build a Pokedex with Platformatic
+
+Recently, [Matteo Collina](https://nodeland.dev/), one of Fastify's creators and much more, launched a new Backend As A Service service: [Platformatic](https://platformatic.dev/).
```
Comment (mcollina) [inline posts/bonus-pokedex.md:3]:
```suggestion
Recently, [Matteo Collina](https://nodeland.dev/), one of Fastify's creators and much more, launched [Platformatic](https://platformatic.dev/): a _fast_ backend development platform.
```

File: posts/bonus-pokedex.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+Since it is built on top of Fastify, it claims to be a life changer, and I want to try it and write down my thoughts!
+
+So, what are we going to build? A Pokedex!
```
Comment (mcollina) [inline posts/bonus-pokedex.md:6]: what’s a pokedex?

File: posts/bonus-pokedex.md [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+
+So, what are we going to build? A Pokedex!  
+Of course, I don't want to write a boring article, let's make it a bit more fun!  
+Here are the requirements:
+
+- The Pokemon database already exists. It will be our "legacy" system
+- I want to build a React frontend. It will test Platformatic's "extensibility". Note that Platformatic is a "Backend" framework, but fastify can handle frontend too!
+- The application must be read-only. I don't want to expose any API with `write` privileges.
+
+Now that we have the requirements let's start!
```
Comment (mcollina) [inline posts/bonus-pokedex.md:14]: A small architecture diagram would be great!

File: posts/bonus-pokedex.md [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+The first step is to create our database schema. Note that this section does not concern Platformatic itself,
+but I want to track my working process.
+
+The following image is the raw Entity-Relation schema for our Pokedex:
+
+![Pokedex DB Schema](./assets/pokedex-er.png)
+
+We must write down the code into a `001.do.schema.sql` file. It will be the first migration file.
+
+Then we need to fill the schema so that we will extract the data from [Poke API](https://pokeapi.co/docs/v2#pokemon).
```
Comment (mcollina) [inline posts/bonus-pokedex.md:28]: You should mention that’ll extract pokemon data in the previous section when you mentioned it was a “legacy” database.

File: posts/bonus-pokedex.md [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+## The database
+
+The first step is to create our database schema. Note that this section does not concern Platformatic itself,
+but I want to track my working process.
+
+The following image is the raw Entity-Relation schema for our Pokedex:
+
+![Pokedex DB Schema](./assets/pokedex-er.png)
+
+We must write down the code into a `001.do.schema.sql` file. It will be the first migration file.
```
Comment (mcollina) [inline posts/bonus-pokedex.md:26]: I think you might want to setup platformatic first, introduce migrations and then fill in the migrations file.

File: posts/bonus-pokedex.md (hunk 3 above)
Comment (Eomm) [reply posts/bonus-pokedex.md]: done

image

Comment (mcollina) [reply posts/bonus-pokedex.md]: is the "React Application Build" a database?

Comment (Eomm) [reply posts/bonus-pokedex.md]: it is the file system - I will try to draw it in a clearer way

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**